### PR TITLE
Add  missing movies and Animes

### PR DESCRIPTION
**Pull request by Arial-Z**
Movies added:
- Demon Slayer -Kimetsu no Yaiba- The Movie: Mugen Train
- Kabaneri of the Iron Fortress: The Battle of Unato
- Made in Abyss: Journey's Dawn
- Made in Abyss: Wandering Twilight
- Made in Abyss: Dawn of the Deep Soul
- My Hero Academia : Two Heroes
- My Hero Academia : Heroes Rising
- The Garden of Sinners (as separate movies)
- The Irregular at Magic High School: The Girl Who Summons the Stars
- Sword Art Online: The Movie – Ordinal Scale

Animes added :
- Ace of Diamond
- Blue Exorcist
- Kamisama Kiss
- Knights of Sidonia
- Major
- Mobile Suit Gundam 00
- Natsume's Book of Friends
- Showa Genroku Rakugo Shinju
- The Demon Girl Next Door
- The Outcast
- The Yakuza's Guide to Babysitting
- Wagnaria!!

Animes synonyms added to match MAL title :
- Darker Than Black > Darker than Black: Kuro no Keiyakusha
- Kaguya-sama: Love is War > Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen
- My Isekai Life: I Gained a Second Character Class and Became the Strongest Sage in the World! > Tensei Kenja no Isekai Life: Dai-2 no Shokugyou wo Ete, Sekai Saikyou ni Narimashita

### CODE REVIEW

**Comment by reconman** [comment]
You were still waiting for review, right? Was pretty busy the last few weeks so I forgot.

**Comment by Arial-Z** [comment]
yeah, but that okay take your time :)

**Comment by Arial-Z** [comment]
Added
- "Gintama: The Movie"
- "Gintama: The Movie: The Final Chapter: Be Forever Yorozuya"
- Added MAL synonym for "KonoSuba: God's Blessing on this Wonderful World! Legend of Crimson"

File: movies-tmdb.en.yaml [hunk 1]
```diff
@@ -18,17 +18,178 @@ entries:
...
 
+  - title: "Kabaneri of the Iron Fortress: The Battle of Unato"
+    synonyms:
+      - "Koutetsujou no Kabaneri Movie 3: Unato Kessen"
+      - "Koutetsujou no Kabaneri: Unato Kessen"
+    seasons:
+      - season: 1
+        anilist-id: 97918
+
+  - title: "Love, Chunibyo & Other Delusions ! -Take On Me"
```
Comment (reconman) [inline movies-tmdb.en.yaml:63]:
```suggestion
  - title: "Love, Chunibyo & Other Delusions! -Take On Me"
```

File: movies-tmdb.en.yaml [hunk 2]
```diff
@@ -18,17 +18,178 @@ entries:
...
+
+  - title: "Made in Abyss: Dawn of the Deep Soul"
+    synonyms:
+      - "Made in Abyss Movie 3: Fukaki Tamashii no Reimei"
+      - "Made in Abyss: Fukaki Tamashii no Reimei"
+    seasons:
+      - season: 1
+        anilist-id: 100643
+
+  - title: "My Hero Academia : Two Heroes"
```
Comment (reconman) [inline movies-tmdb.en.yaml:95]:
```suggestion
  - title: "My Hero Academia: Two Heroes"
```

File: movies-tmdb.en.yaml [hunk 3]
```diff
@@ -18,17 +18,178 @@ entries:
...
+    synonyms:
+      - "Boku no Hero Academia the Movie 1: Futari no Hero"
+      - "Boku no Hero Academia THE MOVIE: Futari no Hero"
+    seasons:
+      - season: 1
+        anilist-id: 100723
+  - title: "My Hero Academia : Heroes Rising"
```
Comment (reconman) [inline movies-tmdb.en.yaml:102]:
```suggestion
  - title: "My Hero Academia: Heroes Rising"
```

**Review by reconman — CHANGES_REQUESTED** [review]
Some incorrect titles with extra spaces.

**Comment by reconman** [comment]
Most of your mappings problably work out-of-the box using PlexAniSync's search function, but I'll add them anyways just in case.

**Comment by Arial-Z** [comment]
Done and thank you

**Review by reconman — APPROVED** [review]

**Pull request merged**